lib/RefBackend/RefBackend.cpp

```diff
@@ -194,7 +194,8 @@ static std::set<std::string> getSupportedConsumeFuncReturnFuncs(OpBuilder &b) {
   Type f64 = b.getF64Type();
 
   SmallVector<TypeRange> supportedReturnTypes = {
-      mri1, mri32, mri64, mrf32, mrf64, i64, f32, f64, {mrf32, mrf32, mrf32}};
+      mri1, mri32, mri64, mrf32,          mrf64,
```

Comment:
Don't forget to do `clang-format`

Reply:
I have done that. This is the resultant formatting.